UCSRest/src/main/java/it/cnr/iit/ucsrest/rest/UCSRestController.java

```diff
@@ -48,42 +48,50 @@
 public class UCSRestController {
     private static final Logger log = Logger.getLogger( UCSRestController.class.getName() );
 
+    enum REQUEST_STATUS {
```

Comment:
is this used?

Reply:
No, it was a leftover, I was trying to add expressiveness to the REST API but Mockito doesn't like it unfortunately